Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile for Pokemon Showdown Server
+FROM node:10
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Copy pacakge and cache things
+COPY package*.json ./
+RUN npm install
+
+# Copy the application into the docker image
+COPY . .
+
+# Expose the server on port 8000
+EXPOSE 8000
+
+# Start the server
+CMD ["node", "pokemon-showdown"]
```

Comment:
You can't just do `["pokemon-showdown"]`? The `node pokemon-showdown` invocation is a Windows-specific workaround because Windows doesn't support Unix-style shell scripts.

Reply:
You can; though I'm not in the habit of prefixing `node` since it works without the hashbang. Looks like you have it in the script, though.